src/utils/fetch-twitch-videos.js

```diff
@@ -0,0 +1,22 @@
+import { get } from './request';
+
+const TWITCH_API_ENDPOINT = 'https://api.twitch.tv/helix/';
```

Comment:
Actually another thought, we might want these in another file altogether since the hooks use them and places that will use `fetchTwitchData` will also need them. Perhaps another constants file makes sense, what do you think?

Reply:
I was actually just thinking about that. It definitely makes more sense to have them in a separate file